src/mono/sample/iOS/Makefile

```diff
@@ -6,7 +6,7 @@ AOT?=false
 TARGET?=iossimulator
 DEPLOY_AND_RUN?=true
 APP_SANDBOX?=false
-STRIP_DEBUG_SYMBOLS?=false # only used when measuring SOD via build-appbundle make target
+STRIP_DEBUG_SYMBOLS?=true # only used when measuring SOD via build-appbundle make target
```

Comment:
I would prefer that the https://github.com/dotnet/runtime/pull/85317/files#diff-425d33169424e0e20bc3bc87ca31fffa5764c9d9eefdcd59f3f6e2c4902da30eR83 specifies explicitly `STRIP_DEBUG_SYMBOLS=true` rather than to have it set by default. 
The reason for this is that stripping symbols also obfuscates the signature and shouldn't be the default behaviour. It is true that symbol stripping is on by default with Xamarin Release builds, but Xamarin also has additional post processing tasks that reapply the signature on a stripped app, making the final app deployable on a device.

Additionally, it might worth considering having 4 measurements:
- MonoAOT with symbols
- MonoAOT without symbols
- NativeAOT with symbols
- NativeAOT without symbols

Although, it might not need to be part of this PR (just a thought).

Reply:
Regarding:
> Additionally, it might worth considering having 4 measurements:

There is a TODO task in https://github.com/dotnet/runtime/issues/80905:
> [ ] Include measuring HelloiOS application size without debug symbols

which should/would be covered by this.

Reply:
Sounds good! There will be the following measurements:
 - MonoAOT llvm with symbols
 - MonoAOT llvm without symbols
 - MonoAOT nollvm with symbols
 - MonoAOT nollvm without symbols
 - NativeAOT with symbols
 - NativeAOT without symbols